plugins/modules/helm.py

```diff
@@ -352,6 +363,14 @@ def run_repo_update(module, command):
     rc, out, err = run_helm(module, repo_update_command)
 
 
+def run_dep_update(module, command, chart_ref):
+    """
+    Run dependency update
+    """
+    dep_update = command + " dependency update " + chart_ref
```

Comment:
this will fail when the ``chart_ref`` contains an URL, you need to check somewhere that it is not the case